domino.go

```diff
@@ -283,6 +317,17 @@ type KeyValue struct {
 /************************************** GetItem ****************************************/
 /***************************************************************************************/
 type get dynamodb.GetItemInput
+type output struct {
+	*dynamodb.GetItemOutput
+	Error error
+}
+
+func (o *output) Result(item interface{}) (err error) {
```

Comment:
Right. We could add other helpers too like, ToInt32 (for single integer returns), or IsEmpty (to test empty returns).